guide/items.md

```diff
@@ -433,4 +433,130 @@ before:
 
 When writing extern items (such as `extern "C" fn`), always be explicit about
 the ABI. For example, write `extern "C" fn foo ...`, not `extern fn foo ...`, or
-`extern "C" { ... }.
+`extern "C" { ... }`.
+
+
+### Imports (`use` statements)
+
+If an import can be formatted on one line, do so. There should be no spaces
+around braces.
+
+```rust
+use a::b::c;
+use a::b::d::*;
+use a::b::{foo, bar, baz};
+```
+
+
+#### Large list imports
+
+Prefer to use multiple imports rather than a multi-line import. However, tools
```

Comment:
What does `prefer` mean here, if tools aren't supposed to enforce it? I thought the focus of these RFCs was to determine how Rust formatting tools behaved.

Reply:
That is the focus, but we have also given advice in some cases for humans to follow where tools can't reformat

Reply:
Sounds good! Thanks for the explanation.